src/components/date-picker/calendar.tsx

```diff
@@ -42,10 +50,18 @@ export class Calendar extends React.Component<CalendarProps, {}> {
                         >
                             <i className="headerArrow headerArrowPrev" />
                         </span>
-                        <span className="headerDate">
-                            <span data-automation-id="MONTH_NAME">
-                                {this.monthName}
-                            </span>
+                        <span
```

Comment:
Is there a reason to create two spans? maybe you can change the text in the span that always renders?